ubuntu-xenial-16.04-cloudimg-console.log

```diff
@@ -0,0 +1,686 @@
+[    0.000000] Initializing cgroup subsys cpuset

```

Comment:
This doesn't look like it belongs here.

Reply:
It totally doesn't, but the log keeps getting added when I look away 😆